Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConcepTypeDeleteController.java

```diff
@@ -62,7 +54,7 @@ public String prepareDeleteType(@PathVariable("typeid") String typeid, ModelMap
         boolean enableDelete = true;
         List<ConceptEntry> conceptEntries = conceptManager.getConceptEntryByTypeId(typeid);
         for (ConceptEntry conceptEntry : conceptEntries) {
-            if ((conceptEntry.getTypeId() != null) && (conceptEntry.getTypeId()).equals(typeid)) {
+            if (conceptEntry.getTypeId() != null) {
```

Comment:
since you've asked for only concepts with a specific type, can the type ever be null?